test/e2e/apps/rc.go

```diff
@@ -45,7 +46,7 @@ var _ = SIGDescribe("ReplicationController", func() {
 		// requires private images
 		framework.SkipUnlessProviderIs("gce", "gke")
 
-		TestReplicationControllerServeImageOrFail(f, "private", "gcr.io/k8s-authenticated-test/serve_hostname:v1.4")
+		TestReplicationControllerServeImageOrFail(f, "private", imageutils.GetE2EImage(imageutils.ServeHostname))
```

Comment:
@ixdy as we already have a way to push these images to user defined repository, can you please run following command to push the images to `gcr.io/k8s-authenticated-test` repository 

`make all-container WHAT=serve-hostname REGISTRY=gcr.io/k8s-authenticated-test`